src/System.ServiceModel.NetNamedPipe/src/System.ServiceModel.NetNamedPipe.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyVersion>$(WcfAssemblyVersion)</AssemblyVersion>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <AssemblyName>System.ServiceModel.NetNamedPipe</AssemblyName>
+    <CLSCompliant>true</CLSCompliant>
+    <IsPackable>true</IsPackable>
+    <IsShipping>$(Ship_WcfPackages)</IsShipping>
+	<!-- TODO: Make this net462 and netstandard2.0 after it's multi-targetting .NET Framework -->
+    <TargetFrameworks>net6.0;net6.0-windows;net461</TargetFrameworks>
```

Comment:
It looks to me like your `net461` implementation is a full facade.  Since you aren't exposing any types on that `TargetFramework` and you don't have a `netstandard2.0` assembly with types that you need to replace with forwards on .NETFramework , could you just delete your `net461` build?

Reply:
FYI, this PR removes any implementation living in System.Private.ServiceModel. We're getting rid of it.  
If I remove the Ref project and build this project for net6.0 and net6.0-windows only, won't that make this package unusable on .NET Framework? I know there are customers who multi target between NetFx and .Net. 

Previously our packages had reference assemblies and implementations for net461, netcoreapp3.1, net60 (needed for MAUI due to Xamarin suppression in our package having higher precedence that netcoreapp3.1), and netstandard2.0.  
With .NET Core 3.1 about to go out of support, there's no need for us to have netstandard2.0. net461 and net6.0 should cover us.

Reply:
@ericstj, are you suggesting the assembly shouldn't be supported on net461? The intention was that the public api's which exist on .NET Framework will type forward to System.ServiceModel on .NET Framework. The idea is that a consumer can multi target to .NET Framework and .NET 6 and use the same package.  

Is that a waste of time and it would be better to have the consumer do the conditional stuff in their project to reference System.ServiceModel when compiling for .NET Framework and only reference the nuget package on .NET 6?  

The reason for not targeting netstandard2.0 is that the implementation would only run on .NET 6+ (as 3.1 is about to go out of support and .NET Framework carries its own) and it frees me up to use newer apis. I'd be fine with a reference assembly targeting netstandard2.0 as I won't be exposing any newer types on the public api surface, just want to consume them in the implementation.

Reply:
> The reason for not targeting netstandard2.0 is that the implementation would only run on .NET 6+ (as 3.1 is about to go out of support and .NET Framework carries its own) and it frees me up to use newer apis. I'd be fine with a reference assembly targeting netstandard2.0 as I won't be exposing any newer types on the public api surface, just want to consume them in the implementation.

I know you were asking Eric for an opinion but let me quickly share mine. These days, with multi-targeting being a first-class concept I wouldn't include a .NETFramework facade assembly just for the sake of a customer not needing to apply a condition when targeting both modern .NET and .NET Framework and referencing this package. We have been doing this in corefx/runtime for a while, but only because we also offer .NETStandard assets. In this case, the package targets modern .NET only, and that's OK and future proof.

Reply:
@ericstj, @ViktorHofer, what are your thoughts on the ServiceModel shim? Its purpose was to allow you to use an assembly compiled for .NET Framework on .NET Core. I'm having problems getting the shim to be generated with the removal of netstandard. Should I put in the work to get that working or drop it? Is that still an important scenario now?

Reply:
IMHO I would drop it as running .NET Framework compiled assemblies on modern .NET was never a "first class" supported scenario. When you do a "dotnet add reference" pointing to a .NET Framework project you will even get an error today:

```cmd
C:\temp\modernent>dotnet add reference ..\netfx\
Project `C:\temp\netfx\netfx.csproj` cannot be added due to incompatible targeted frameworks between the two projects. Review the project you are trying to add and verify that is compatible with the following targets:
    - net7.0
```